src/core/interaction_data.hpp

```diff
@@ -921,6 +953,36 @@ int virtual_set_params(int bond_type);
 void set_dipolar_method_local(DipolarInteraction method);
 #endif
 
+//Checks if there is a specific bond between p_bond and p_partner, where the bond is stored on p_bond. Needs GHOSTS_HAVE_BONDS if particles are ghosts.
+inline bool are_bonded(const Particle * const p_bond, const Particle * const p_partner, BondedInteraction bond)
```

Comment:
why dont you name it check_if_bonded or similar?

Reply:
Why not `is_bonded`???  Those are the important questions!

Reply:
I will call them 
`check_first_particle_for_bond_between_particles`
and 
`check_for_bond_between_particles`